src/components/App/MenuContainer/NavigationMenuList/NavigationMenuList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import MenuItem from './NavigationMenuItem/NavigationMenuItem';
+import { StyledNavMenu } from './NavigationMenuListStyles';
+import { ItemsInterface, NestedMenuType } from '../types/types';
+import { BrowserRouter } from 'react-router-dom';
+
+export interface Props {
+  menuItems: ItemsInterface[];
+  nestedRoute?: string;
+  addNestedMenu?(elem: NestedMenuType): void;
+}
+
+const NavigationMenu: React.FC<Props> = (props: Props) => {
+  const [active, setActive] = useState<boolean>(false);
```

Comment:
You've already have default value, remove \<boolean\>

Reply:
ok